sql/select.go

```diff
@@ -402,8 +399,13 @@ func (v *indexInfo) makeConstraints(exprs []parser.Exprs) {
 				if !startDone && constraint.start == nil {
 					switch c.Operator {
 					case parser.GT:
-						startDone = true
-						fallthrough
+						// Transform a > constraint into a >= constraint so that we play
```

Comment:
It would seem more natural to perform this transformation in `simplifyComparisonExpr` and yet performing it there causes some of the subsequent simplifications of expressions to no longer occur. I've got a TODO to reexamine this.